assets/install-rhcos.sh

```
@@ -1,53 +1,145 @@
+            ilog "INFO: Found file sized $(stat -c %s ${cfg_file}) bytes, extracting ignition config..."
+            cat ${cfg_file} | tr -d '\0' > $IGNITION
+        fi
+        
+        rm -f $cfg_file
+    fi
+}
+
+install_rhcos() {
+    default_device=/dev/mmcblk0
+    if [ -b /dev/nvme0n1 ]; then
+        default_device="/dev/$(cd /sys/block; /bin/ls -1d nvme* | sort -n | tail -1)"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)